misk/src/main/kotlin/misk/web/actions/StatusActionApi.kt

```diff
@@ -0,0 +1,34 @@
+package misk.web.actions
+
+import misk.healthchecks.HealthCheck
+import misk.web.Get
+import misk.web.ResponseContentType
+import misk.web.mediatype.MediaTypes
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class StatusActionApi : WebAction {
+    @Inject lateinit var healthChecks: MutableSet<HealthCheck>
+
+    @Get("/api/_status")
+    @ResponseContentType(MediaTypes.APPLICATION_JSON)
+    fun statusActionApi(): Status {
```

Comment:
nit: action methods should be named consistently with regular methods - as verbs describing what they do (.e.g `getStatus`)